components/SupportUs/SupportUs.tsx

```diff
@@ -0,0 +1,83 @@
+import { items } from './SupportItemsList';
+
+interface SupportUsProps {
+  className: string;
+  showSupportBanner: boolean;
+};
+
+/**
+ * This component displays support items.
+ * @param {SupportUsProps} props - The props for SupportUs component.
+ * @param {string} props.className - Additional CSS classes for styling.
+ * @param {boolean} props.showSupportBanner - Indicates whether support banner should be displayed.
+ */
+export default function SupportUs({
+  className = '',
+  // eslint-disable-next-line no-unused-vars, unused-imports/no-unused-vars
+  showSupportBanner = true
```

Comment:
You can remove this prop if it is not being used anywhere inside the component, instead of disabling the eslint.

Reply:
Done